ckanext/datajson/package2pod.py

```diff
@@ -146,7 +146,12 @@ def export_map_fields(package, json_export_map, redaction_enabled=False):
                     method = getattr(Wrappers, wrapper)
                     if method:
                         Wrappers.current_field_map = field_map
-                        dataset[key] = method(dataset.get(key))
+                        try:
+                            dataset[key] = method(dataset.get(key))
+                        except TypeError as err:
+                            log.warning('Error with package %s when exporting field %s: %s', package.get('id'), key, err,
+                                exc_info=sys.exc_info())
```

Comment:
looks like lint didn't like this line: E128 continuation line under-indented for visual indent